test/libraries/Pool.t.sol

```diff
@@ -119,9 +119,12 @@ contract PoolTest is Test {
         assertEq(slot0.protocolFee(), 0);
         slot0 = slot0.setProtocolFee(protocolFee);
         assertEq(slot0.protocolFee(), protocolFee);
+        state.slot0 = slot0;
+
+        uint16 pFee = params.zeroForOne ? protocolFee0 : protocolFee1;
```

Comment:
i dont like `pFee` 😆 can we rename it something more descriptive? Or rename `protocolFee` and call this one `protocolFee`?

Reply:
swapProtocolFee? finalProtocolFee? expectedProtocolFee?